utils.py

```diff
@@ -150,3 +151,6 @@ def Make1DHists(truth, predict, PUPPI, xmin=0, xmax=400, nbins=100, density=Fals
     plt.ylabel(yname)
     plt.savefig(outputname)
     plt.close()
+    
+def to_np_array(ak_array, maxN=100, pad=0):
```

Comment:
If you define this here, is it still defined in `convert.py`? If so, you should remove it from there and import it from here (no code duplication)